terraform/task-runner/README.md

```diff
@@ -0,0 +1,49 @@
+# Task runner
+
+This project enables developers to run manual and scheduled tasks in ECS.
+
+## Running a rake task
+
+To run a one-off rake task, you must do something like the following:
+
+Create a task definition using the terraform project (e.g. frontend app)
+
+Run the task, with an override for the command:
+
+```sh
+aws ecs run-task --cluster task_runner --task-definition frontend \
+--launch-type FARGATE --count 1 --started-by "Harry Potter" \
+--network-configuration '{
+  "awsvpcConfiguration": {
+    "subnets": ["subnet-6dc4370b", "subnet-463bfd0e", "subnet-bfecd0e4"],
+    "securityGroups": ["sg-0b873470482f6232d"],
+    "assignPublicIp": "DISABLED"
+  }
+}' \
+--overrides '{
+  "containerOverrides": [{
+    "name": "frontend",
+    "command": ["/bin/bash", "-c", "bundle exec rake stats || exit 1"]
```

Comment:
I think this could be simpler as:

```suggestion
    "command": ["bundle", "exec", "rake", "stats"]
```

Reply:
Use `bash -c` at your peril

https://hynek.me/articles/docker-signals/

Reply:
Thanks both! That's a helpful blog post.